mobile-app/lib/service/podcast/notification_service.dart

```diff
@@ -34,12 +34,12 @@ class NotificationService {
       iOS: iosInitializationSettings,
     );
 
+    await _flutterLocalNotificationsPlugin.initialize(initializationSettings);
+
     await _flutterLocalNotificationsPlugin
```

Comment:
Could you put this part behind an if block that checks if the mobile OS is android or not